spotifyutils/__init__.py

```diff
@@ -49,8 +53,4 @@ def cli(input_args=None):
     )
 
     args = parser.parse_args(input_args)
-    main(**vars(args))
-
-
-def main(**kwargs):
-    print('Test')
+    configuration(**vars(args))
```

Comment:
This function currently returns a tuple of (access token, refresh token), but those values are unused. Refer to the related comment.